src/styles/components/scrollbar/styles.less

```diff
@@ -67,6 +67,7 @@
     position: relative;
 
     .gm-scroll-view {
+      display: flex;
```

Comment:
this is a pretty big change, are we sure that is doesnt break something else? 😨

Reply:
You can never be sure, but I tried my best to verify that nothing broke